config/utils/getIsOnMobileScreenSize.ts

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+import { breakpoints } from "../themes/screen";
+import {
+  getCurrMediaWidth,
+  useEffectOnScreenResize,
+} from "./useEffectOnScreenResize";
+
+export function getIsOnMobileScreenSize(): boolean {
```

Comment:
I would recommend we make changes that address the following:
1. This function named with `get` prefix but it also sets values. Can we rename it something else? Maybe separate setter from getter?
2. Let's avoid the name `mobile`. I think it is an industry term that we should move away because of its ambiguity. mobile could mean ipad, tablet, etc... are also mobile. Can we use `smallScreen` or even better, the breakpoint `small`